recipes-support/ostree-pending-reboot/ostree-pending-reboot_0.1.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "OStree Pending Reboot service"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit allarch systemd
+
+RDEPENDS_${PN} = "aktualizr-get" # TODO: ??
+
+SRC_URI = "file://ostree-pending-reboot.service"
+
+S = "${WORKDIR}"
+
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+SYSTEMD_SERVICE_${PN} = "ostree-pending-reboot.service"
+SYSTEMD_AUTO_ENABLE_${PN} = "enable"
```

Comment:
So this service was initially created to be used together with a systemd timer job, so it shouldn't really be started/enabled automatically (as it will not even start due ConditionPathExists being false during boot).

You could have a timer created and installer by this recipe as well, and enable that one by default (or via configs that can be set by the user).

Reply:
I think we wanted this to be enabled / started by default if the user image chooses it.  So I'll get the timer portion in as well here.

Reply:
What you want is the timer to be enabled by default, the reboot.service one doesn't require that as it gets triggered by the timer job.